host-controller/src/main/resources/org/jboss/as/domain/controller/resources/LocalDescriptions.properties

```diff
@@ -43,12 +43,12 @@ domain.start-servers.blocking=Wait until the servers are fully started before re
 domain.start-servers.start-mode=The mode the servers should start running in, can be either suspend or normal.
 domain.stop-servers=Stops all servers currently running in the domain.
 domain.stop-servers.blocking=Wait until the servers are stopped before returning from the operation.
-domain.stop-servers.suspend-timeout=The graceful shutdown timeout. If this is zero then a graceful shutdown will not be attempted, if this is -1 then the server will wait for a graceful shutdown indefinitely.
+domain.stop-servers.suspend-timeout=The graceful shutdown timeout in seconds. If this is zero (the default) then the server will shutdown immediately. A value larger than zero means the server will wait up to this many seconds for all active requests to finish. A value smaller than zero means that the server will wait indefinitely for all active requests to finish.
```

Comment:
Shall we prefer to use `stop` word instead of `shutdown` in this case?

Reply:
It makes sense to me, changed, thanks